pkgs/build-support/build-dotnet-module/hooks/dotnet-build-hook.sh

```diff
@@ -0,0 +1,37 @@
+declare -a projectFile testProjectFile dotnetBuildFlags dotnetFlags
+
+dotnetBuildHook() {
+    echo "Executing dotnetBuildHook"
+
+    runHook preBuild
+
+    if [ ! -z "${enableParallelBuilding-}" ]; then
+        maxCpuFlag="$NIX_BUILD_CORES"
+        parallelBuildFlag="true"
```

Comment:
Do we local those variables in other hooks?

Reply:
`$maxCpuFlag` is used in `dotnet-check-hook`, and the value stays the same. The second one only seems to be used while actually building.